libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -132,6 +132,17 @@ AP_BLHeli::AP_BLHeli(void)
     last_control_port = -1;
 }
 
+// initialize
+void AP_BLHeli::init(void)
+{
+    // filter the motor rpm at half the sample rate
+    // if the frequency requested of the harmonic notch is not smooth then the
+    // resulting changes look like gyro noise
+    for (uint8_t i = 0; i < max_motors; i++) {
+        motor_rpm[i].set_cutoff_frequency(telem_rate.get(), telem_rate.get() * 0.48f);
```

Comment:
It does not make sense to filter the frequency output at half the update rate. This will provide next to no smoothing. I will need to chat to you off line.

Reply:
Fixed